Spanish translation

wmlgr opened the pull request:
I want to help translate this great extension into Spanish.

martinkadlec0 commented:
Hey, thanks for the effort but this extension is not maintained for multiple years now. Honestly, I am surprised anyone is still using it as I heard it is broken in lots of ways. That said, I am fine with anyone forking and even publishing again it if you want to.

miso-belica commented:
@martinkadlec0 What do you think about adding someone the perms to this repo who can setup GitHub action to publish it and maintain the PRs, ...? So the work will not be split and the people get the latest updates automatically.

I use different RSS for a year or so now, but maybe @wmlgr or someone else is interested. Just an idea :)

zakius committed:
Add rendering of only unread sources when selected

zakius committed:
Add `show only sources with unread articles` to settings interface

zakius committed:
Add button toggling `show only sources with unread` to toolbar

closes #148

zakius committed:
Add `remove selected source` button to toolbar

closes #146

zakius committed:
Add `scroll selected source/folder into view` button to toolbar

closes #147

zakius committed:
Fix auto removing from trash

zakius committed:
Reduce scope of analysed requests when removing CSP

zakius committed:
Fix CSP handler not being applied when needed

zakius committed:
Add setting to choose favicon provider

zakius committed:
Clean up

zakius committed:
Add favicon source to options interface

zakius committed:
auto version bump: minor

zakius committed:
Formatting

zakius committed:
Add ResizeObserver

closes #151

zakius committed:
Add rejection to checkFavicon if getFaviconAddress fails

closes #150

zakius committed:
cleanup

zakius committed:
Fix special string

closes #127

zakius committed:
Formatting

nijssen committed:
Fix RSSParser to handle relative links

nijssen committed:
auto version bump: patch

OscarL committed:
Added a quick translation into Spanish

zakius committed:
Add message handlers

zakius committed:
Fix loading of folder properties

zakius committed:
Merge pull request #156 from nijssen/fix-rssparser-relative-links

Fix bug in RSSParser where feed relative links are not properly handled

zakius committed:
Merge pull request #157 from OscarL/patch-1

Added a quick translation into Spanish

zakius committed:
Fix handling of relative links

zakius committed:
Merge branch 'master' of https://github.com/SmartRSS/Smart-RSS

zakius committed:
Fix pinned counter not updating when removing pinned items

closes #153

zakius committed:
Add parsing for prefixed tags

references #159

zakius committed:
Enable context menu for all inputs

references #160

zakius committed:
Fix favicon parsing

references #160

zakius committed:
Cleanup favicon parsing

zakius committed:
Cleanup favicon parsing

zakius committed:
Improve favicon parsing:

- Attempt to load all found links + default and use first one that works

- Include Promise.any polyfill for older browsers

zakius committed:
auto version bump: patch

zakius committed:
Add Spanish to list

closes #162

zakius committed:
Fix date check

references #127

zakius committed:
Cleanup

zakius committed:
Cleanup

consistency with trash autoremoval

zakius committed:
auto version bump: patch

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Switch to simplified CommonJS wrapping

zakius committed:
Cleanup

zakius committed:
Add DST offset handling to Day of Year

zakius committed:
Add DST offset handling to Week of Year

zakius committed:
Extract the method to unify

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Switch to simplified CommonJS wrapper

zakius committed:
Cleanup

zakius committed:
Update Backbone

zakius committed:
Extract variable declaration

zakius committed:
Add styles for dark theme

zakius committed:
Add invertColors to Settings

zakius committed:
Add invertColors to Options

zakius committed:
Cleanup

zakius committed:
Add applying inverted theme

zakius committed:
Cleanup

zakius committed:
Cleanup

zakius committed:
Apply styles on app init

zakius committed:
Apply styles on app init

zakius committed:
Cleanup

zakius committed:
Cleanup

zakius committed:
Rename variable

zakius committed:
Add workaround for spidersweb.pl youtube embed being broken in RSS

zakius committed:
Replace youtube enclosure with nocookie variant

p0k33m0n committed:
Update pl.js

Updated transaltion for Polish language.

Cannot find strings: 
1) scroll into view
2) Toggle show only unread

zakius committed:
Allow view opening when store is not yet initialized

register BrowserAction as soon as possible

includes animation by [DitherSky](https://github.com/Dither)

zakius committed:
Add notice about backing up feeds

zakius committed:
Remove excessive he decoding

closes #178 but may require further observation

zakius committed:
Fix header link

closes #179

zakius committed:
Fix article changes detection and update more fields

closes #172

zakius committed:
Handle Youtube newlines and fix video id parsing

closes #168

zakius committed:
Loads placeholder for youtube videos

Uses modified version of https://github.com/paulirish/lite-youtube-embed to achieve this

References #168

zakius committed:
Merge pull request #164 from p0k33m0n/patch-1

Update pl.js

zakius committed:
cleanup

zakius committed:
Merge remote-tracking branch 'origin/master'

zakius committed:
Move styles to stylesheet

zakius committed:
cleanup

zakius committed:
Change esversion

zakius committed:
Add defaultView setting

zakius committed:
Add defaultView setting to sources

zakius committed:
Add defaultView setting to sources

zakius committed:
Add defaultView to properties

zakius committed:
Add defaultView to options

zakius committed:
Add Mozilla Readability

zakius committed:
Add change view button

zakius committed:
Cleanup

zakius committed:
Add setting for caching articles after parsing

zakius committed:
Add parsedContent property

zakius committed:
Clean parsedContent property when article updates

zakius committed:
Add caching support to content view

zakius committed:
Add caching support to content view

zakius committed:
Add article caching setting in the interface

zakius committed:
Add check to not save parsed content when already loaded from cache

zakius committed:
Add support of multiple enclosures to content view

zakius committed:
Change default value of enclosure field

zakius committed:
Add support of multiple enclosures to feed parser

closes #174

zakius committed:
cleanup

zakius committed:
Attempt to fix issue causing items to be inserted out of order if feed update happens when articles list is displayed

zakius committed:
Remove improper existence checks

closes #166

zakius committed:
fix formatting

zakius committed:
Add setting to show only unread articles by default

zakius committed:
Reverse behavior if defaulting to unread only

zakius committed:
Add setting

zakius committed:
Hide for now

zakius committed:
cleanup

zakius committed:
Add configurable hotkeys + groundwork for localization of settings

closes #10 though still needs some polishing, especially adding many new strings

zakius committed:
Don't enforce checking own property

zakius committed:
auto version bump: minor

zakius committed:
Don't compare enclosures

zakius committed:
auto version bump: patch

zakius committed:
Fix issue causing content view to break in certain conditions

closes #182

zakius committed:
Change change detection

- ignore dates

- if content seems to be HTML extract `innerText` to skip identifiers that may be re-generated on load

closes #180

closes #181

zakius committed:
auto version bump: patch

zakius committed:
Add `base` to content document

closes #185

zakius committed:
Remove extension address from parsed view

closes #186

zakius committed:
cleanup

zakius committed:
Fix cached content

zakius committed:
Fix cached content

zakius committed:
closes #184

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
Localize more actions

zakius committed:
Fix space pt.1

zakius committed:
Fix space pt.2

zakius committed:
Fix space pt.3

zakius committed:
Fix keyboard navigation in sources pane

zakius committed:
Deduplicate code

zakius committed:
Change sources list to `a` elements:

- handles scrolling into view on focus
- allows middle-click to open homepage

zakius committed:
Scroll freshly added source into view

zakius committed:
Handle lack of property

zakius committed:
Handle lack of property

zakius committed:
Handle lack of property

zakius committed:
Only use direct children for homepage detection

zakius committed:
Mark active region

zakius committed:
Fix focus after emptying articles list

zakius committed:
Rename options entrypoint

zakius committed:
Rename app entrypoint

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
Update deprecated call

zakius committed:
cleanup

zakius committed:
force clear

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
always display status bar

zakius committed:
move to messages

zakius committed:
fix button display

zakius committed:
polish statusbar

zakius committed:
cleanup

zakius committed:
Make date groups sticky

zakius committed:
Refactor date handling

zakius committed:
cleanup

zakius committed:
Disable parsed content caching

zakius committed:
Use mutation observer on YouTube only

zakius committed:
Remove unused event

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
Load templates JIT

zakius committed:
Rename

zakius committed:
Update and strip down require.text.js to contain only code necessary for the extension to work

Add license and copyright info

zakius committed:
Minify readability.js

zakius committed:
Minify backbone.min.js

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
Prevent rendering if model changed

zakius committed:
cleanup

zakius committed:
flip condition

zakius committed:
add `id` tag parsing when generating unique item id

zakius committed:
Revert "flip condition"

This reverts commit 3c64aa7423bb9fda71364b76c48537707beca591.

zakius committed:
fix issue preventing rendering items list with specific date format set

zakius committed:
add style for pressed buttons

zakius committed:
fix youtube descriptions formatting

zakius committed:
fix old article recognition

zakius committed:
improve article list rendering

zakius committed:
auto version bump: patch

zakius committed:
Fix search

closes #200

zakius committed:
Dirty fix for title clipping

closes #199

zakius committed:
Fix issue preventing from saving properties of a folder

references #196

zakius committed:
Fix content comparison

zakius committed:
Refactor settings to use booleans

zakius committed:
Refactor item properties to make process of getting global values cleaner

zakius committed:
Change to newer API

zakius committed:
Fix settings

zakius committed:
Fix settings

zakius committed:
cleanup

zakius committed:
Fix article change detection

zakius committed:
auto version bump: patch

zakius committed:
Add detection of GitHub releases feeds

closes #205

zakius committed:
Responsive mode WIP

zakius committed:
auto version bump: minor

zakius committed:
cleanup

zakius committed:
Add support for toggle buttons

zakius committed:
Add support of Unicode glyphs to toolbar buttons

Change `toggle show only unread` button icon

closes #193

zakius committed:
Add option to disable link context menu

closes #189

zakius committed:
cleanup

zakius committed:
Slight improvement in favicon display

references #192

zakius committed:
Add support of browser notifications, WIP

todo: translate message

references #187

zakius committed:
cleanup

zakius committed:
Improve handling of alternative clicks on source

zakius committed:
Add setting to GUI

zakius committed:
Add action

zakius committed:
Add `display only unread articles` button to articles list

closes #155

zakius committed:
Fix YouTube playlist id matching

closes #206

zakius committed:
Add type definitions

zakius committed:
Update deprecated call

zakius committed:
Remove unused property

zakius committed:
Cleanup

zakius committed:
Add Mark all as read

zakius committed:
Add store for per-origin throttling

zakius committed:
Add per-domain throttling and updating URL according to redirect

zakius committed:
Cleanup

zakius committed:
auto version bump: minor

zakius committed:
Cleanup

zakius committed:
Push throttled source at the end of queue

zakius committed:
cleanup

zakius committed:
Fix improper handling of boolean settings

zakius committed:
Fix not being able to load autoremoval behavior

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
cleanup

zakius committed:
auto version bump: patch

zakius committed:
Improve favicon handling

Only domain-throttle for Openrss for now

zakius committed:
Migrate from Chrome to Browser APIs

Refactor extension to use cross-browser WebExtensions browser.* APIs instead of chrome.* APIs, improving compatibility with Firefox and other browsers. Key changes include:
- Replaced chrome.runtime with browser.runtime
- Updated browser action and notification methods
- Adjusted API calls in background scripts and views
- Removed Chrome-specific manifest entries

zakius committed:
auto version bump: minor

zakius committed:
Update README.md

zakius committed:
Migrate build system from Grunt to Node.js

Replace Grunt-based build process with a custom Node.js build script, simplifying project setup and build tasks. Key changes include:
- Remove Grunt and related dependencies
- Add npm scripts for build, package, and release tasks
- Update README with new build system instructions
- Bump minimum Node.js version to 12.0.0
- Add chokidar for file watching

zakius committed:
Modernize JavaScript Code Quality and Style

Introduce ESLint for improved code quality and consistency:
- Add ESLint configuration
- Update .editorconfig with JS-specific quote style
- Refactor code to use modern JavaScript practices
- Replace deprecated hasOwnProperty with hasOwn
- Simplify null/undefined checks
- Improve code readability and reduce potential bugs
- Update package.json with lint script

zakius committed:
Replace hasOwn with hasOwnProperty in library files

Standardize method usage across library files:
- Update require.js to use hasOwnProperty
- Update jquery.min.js to use hasOwnProperty
- Update underscore.min.js to use hasOwnProperty
- Ensure consistent method naming in third-party libraries

zakius committed:
Update README.md

zakius committed:
Update README.md
